elytron/src/main/java/org/wildfly/extension/elytron/ElytronExtension.java

```diff
@@ -145,6 +147,7 @@ public void initializeParsers(ExtensionParsingContext context) {
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_16_0, () -> new ElytronSubsystemParser16_0());
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_17_0, () -> new ElytronSubsystemParser17_0());
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_18_0, () -> new ElytronSubsystemParser18_0());
+        context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_COMMUNITY_18_0, () -> new ElytronSubsystemParserCommunity18_0());
```

Comment:
If your ElytronSubsystemSchema enumeration implements getXMLDescription() correctly, then schema registration for all versions becomes a one-liner, e.g.

```java
context.setSubsystemXMLMappings(SUBSYSTEM_NAME, EnumSet.allOf(ElytronSubsystemSchema.class));
```

Reply:
@pferraro Thank you very much for quick review! It all sounds good! I am working on all these changes now

Reply:
@pferraro Thanks! I now use `context.setSubsystemXmlMappings(SUBSYSTEM_NAME, EnumSet.allOf(ElytronSubsystemSchema.class));` as you suggested